plugins/basename/Dockerfile

```diff
@@ -1,20 +1,20 @@
-FROM rapid7/insightconnect-python-3-38-slim-plugin:4
-LABEL organization=komand
+FROM rapid7/insightconnect-python-3-38-plugin:5
+
+LABEL organization=rapid7
 LABEL sdk=python
-LABEL type=plugin
 
-ENV SSL_CERT_FILE /etc/ssl/certs/ca-certificates.crt
-ENV SSL_CERT_DIR /etc/ssl/certs
-ENV REQUESTS_CA_BUNDLE  /etc/ssl/certs/ca-certificates.crt
+WORKDIR /python/src
 
```

Comment:
Do we wanna preserve these `ENV` variables? I noticed refresh tends to wipe Dockerfiles.

Reply:
Unsure. I'll import and test as a custom plugin. This plugin makes no requests.

Reply:
Removal doesn't appear to cause any issues.